recipes/psrchive/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "psrchive" %}
+{% set version = "2021-01-18" %}
+
+package:
+  name: "{{ name | lower }}"
+  version: "{{ version | replace('-','.') }}"
+
+source:
+  # e.g., https://sourceforge.net/projects/psrchive/files/psrchive/2021-01-18/psrchive-2021-01-18.tar.gz/download
+  url: https://sourceforge.net/projects/{{ name }}/files/{{ name }}/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 74b2bd483bb8928bd070b96e7c845e1678213ba57bbdfa5a63726635554f8e9d
+  patches:
+    - readline.patch
+        
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - make
+    - perl
+    - autoconf
+    - automake
+    - libtool
+    - swig
+  host:
+    - cfitsio
+    - pgplot
+    - gsl
+    - python {{ python }}
+    - numpy
+    - fftw
+    - tempo2
+    - libcblas * *netlib
```

Comment:
I'm not 100% sure on the syntax here. I thought it should be *=*. @isuruf, sorry to bother you with this but can you take a quick look?

Reply:
Just `libcblas`

Reply:
So this was a tricky/confusing issue for me.  The package does not directly depend on BLAS, only indirectly via GSL (it uses some of the `gsl_blas_*` functions).  Here are the various things I tried and what the results were:

1. Only `gsl` in the requirements (ie no BLAS library explicitly mentioned).  This resulted in openblas being installed during the build, and an "overlinking" error for libopenblas.
2. `libcblas` added to requirements.  Same results as above (build used openblas, overlinking error).
3. `libcblas * *netlib` in the requirements (host section only).  This resulted in the netlib BLAS being used successfully in the build, with no errors.  The test stage used openblas with no problem.

I arrived at this final version after reading https://conda-forge.org/docs/maintainer/knowledge_base.html#blas which implied builds should use netlib, although this didn't happen until I explicitly specified it.

All the results from these various attempts can be found in the failed builds above in this PR but please let me know if there is other specific information I can provide regarding this.  Suggestions on how best to solve this are certaintly welcome!

Reply:
Hi folks, any additional thoughts on this?  TL;DR version of my last post was that I tried "just `libcblas`" and it didn't work.  Thanks!

Reply:
> All the results from these various attempts can be found in the failed builds above in this PR but please let me know if there is other specific information I can provide regarding this. Suggestions on how best to solve this are certaintly welcome!

Sorry I did not checked the commit history. Did you try Isuru's suggestion and that did not work? If not, do you mind testing it so we can check?

Reply:
Yes, I did try that and it did not work (it was "#2" in my list above).  But it looks like I only did that test on my local system.  I'll push that change here so you guys can take a look.

Reply:
> I'll push that change here so you guys can take a look.

Thanks. If there is nothing we can do to fix it we can revert and merge as-is.

Reply:
Glad this worked, thanks again for the help!  I'll have to look into why it failed for me locally, maybe different version of conda build or something like that.